src/components/HURUmap/ExploreButton/index.js

```diff
@@ -0,0 +1,49 @@
+import ToggleButton from "@material-ui/lab/ToggleButton";
+import ToggleButtonGroup from "@material-ui/lab/ToggleButtonGroup";
+import Image from "next/image";
+import PropTypes from "prop-types";
+import React, { useState } from "react";
+
+import useStyles from "./useStyles";
+
+const TreeView = ({ items, ...props }) => {
```

Comment:
1. Why TreeView?
2. While it's still up for [debate](https://stackoverflow.com/questions/49306148/why-is-arrow-syntax-preferred-over-function-declaration-for-functional-react-com), I'd rather we use normal `function` at component level.

Reply:
1. just a sheer wrong name. I had fixed it but hadn't pushed.